packages/central-server/src/utilities/sendEmail.js

```diff
@@ -10,6 +10,10 @@ const DEFAULT_SIGN_OFF = 'Cheers,\n\nThe Tupaia Team';
 export const sendEmail = async (to, subject, text, attachments, signOff = DEFAULT_SIGN_OFF) => {
   const { SMTP_HOST, SMTP_USER, SMTP_PASSWORD, SITE_EMAIL_ADDRESS } = process.env;
 
+  if ([SMTP_HOST, SMTP_USER, SMTP_PASSWORD, SITE_EMAIL_ADDRESS].includes(undefined)) {
+    return {};
+  }
```

Comment:
```suggestion
  if (!SMTP_HOST || !SMTP_USER || !SMTP_PASSWORD || !SITE_EMAIL_ADDRESS) {
    return;
  }
```

Pretty minor suggestion here. I guess I find it a bit more readable this way? But no pressure.

Also for the return value, I'm almost certain we actually never use the return value of this function, so I think it's safe to just return nothing. 👍

Reply:
Yep good suggestion, will apply that. This function expects to return a value, so might just keep the `return {};`

Reply:
I think the return value is just a `promise` to be `awaited`, which this will return regardless of whether we return a value or not due to it being an `async` function. But yes, I'm just being pedantic, I'm fine with whatever 👍

Reply:
Oh not at all, thanks for sharing! I did try to use `return;`, but Eslint gets suspicious to say that this function should either return value or not. 🫠

Reply:
Ah, thanks for trying 🙏 I daren't question Eslint's guidance 😁